src/FsAutoComplete.Core/AbstractClassStubGenerator.fs

```diff
@@ -102,7 +80,16 @@ let tryFindAbstractClassExprInBufferAtPos
   =
   asyncOption {
     let! parseResults = codeGenService.ParseFileInProject document.FileName
-    return! tryFindAbstractClassExprInParsedInput pos parseResults.ParseTree
+
+    return!
+      (pos, parseResults.ParseTree)
+      ||> ParsedInput.tryPick (fun _path node ->
+        match node with
+        | SyntaxNode.SynExpr(SynExpr.ObjExpr(
+            objType = baseTy; withKeyword = withKeyword; bindings = bindings; newExprRange = newExprRange)) ->
+          Some(ObjExpr(baseTy, bindings, newExprRange, withKeyword))
+        | SyntaxNode.SynModule(SynModuleDecl.Types(types, _)) -> List.tryPick walkTypeDefn types
```

Comment:
most of the other refactorings I see in this PR didn't need to manually walk the innards of the DU case quite like this - is this a sign that the `types` in a `SynModule` aren't walked in the 'default' walker?

Reply:
Hmm, I think in this particular case it's just that I didn't really look into the implementation of `walkTypeDefn` to see whether it could be inlined and simplified. Glancing at it now, it looks like it probably does actually need to do what it's doing, although I haven't looked into how `AbstractClassData.ExplicitImpl` is ultimately used.

Reply:
...Oh, never mind, I see what you mean.

Reply:
498234d